src/Spd.Manager.Licence/SharedContract.cs

```diff
@@ -196,4 +198,12 @@ public record ContactInfo
     public string? MiddleName1 { get; set; }
     public string? MiddleName2 { get; set; }
     public string? Surname { get; set; }
+}
+
+public record LicenceAppBase

```

Comment:
put this definition to LicenceAppContract.cs

Reply:
Fixed